src/simplify_dyn_ops.cpp

```diff
@@ -328,6 +464,7 @@ void simplify_dyn_ops::apply(module& m) const
                         find_const_3in_slice{},
                         find_const_4in_slice{},
                         find_const_alloc_fill{});
+    match::find_matches(m, simplify_select_module_output_shape{});
```

Comment:
Does adding a second find_matches() call have a different effect than adding the new pass at the end of the preceding list?

Reply:
Yes, tuns the matcher after running all of the prior through the model